docs/proposals/formatted-output.md

```diff
@@ -75,6 +75,14 @@ jobs:
           cat $PATH
 ```
 
+### Verify local files
+
+Carol is a build engineer who needs to check whether some locally copied files are identical to the remote artifact content. she can utilize `oras manifest fetch` to generate a checksum file based on the manifest without pulling the whole artifact. (See related issue [here](https://github.com/oras-project/oras/issues/1368)).
+
+```bash
+oras manifest fetch $REGISTRY/$REPO:$TAG --format go-template='{{range .content.layers}}{{if index .annotations "org.opencontainers.image.title"}}{{(split ":" .digest)._1}}  {{index .annotations "org.opencontainers.image.title"}}{{println}}{{end}}{{end}}' | shasum -c
```

Comment:
What's the default algorithm for `shasum`? Should we set it to `shasum -a 256`?

Reply:
From usage perspective, `shasum` detects algorithm based on the length of the sum, e.g. it can check a file containing both sha256 and sha512 check sum
```console
> echo 256 > 256
> echo 512 > 512
> shasum -a 256 256 >> checksum
> shasum -a 512 512 >> checksum
> cat checksum
25bfed1cc9e0857d2ba433f97e50d92204946c3deac5adfbb917afa39d5132b18ec8da90352dec41c089962f83a18cbfd7174f5fcabc5474d96c949b7d11e324  512
f16c302d5d30e1d3fbe955cf4f637f58a871adeba597922e3baad0aeeb13f656  256
> shasum -c checksum
512: OK
256: OK
```